h/groups/__init__.py

```diff
@@ -1,5 +1,9 @@
 # -*- coding: utf-8 -*-
 
+from h.groups.auth import annotation_group_write_permitted
+
 
 def includeme(config):
+    config.set_memex_group_write_permitted(annotation_group_write_permitted)
```

Comment:
I'd suggest testing the `includeme` with a stub configurator (as we do, for example, in `routes_test.py`).